src/jit/gentree.h

```diff
@@ -4602,14 +4595,11 @@ struct GenTreePutArgStk : public GenTreeUnOp
     // TODO-Throughput: The following information should be obtained from the child
     // block node.
 
-    enum PutArgStkKind : __int8{
-        PutArgStkKindInvalid, PutArgStkKindRepInstr, PutArgStkKindUnroll,
-    };
+    enum PutArgStkKind : __int8{PutArgStkKindInvalid, PutArgStkKindRepInstr, PutArgStkKindUnroll};
 
     PutArgStkKind gtPutArgStkKind;
 
     unsigned gtNumSlots;             // Number of slots for the argument to be passed on stack
-    bool     gtIsStruct;             // This stack arg is a struct.
```

Comment:
Thanks for getting rid of this. I think we might also want to consider eliminating the other fields, as they can be obtained from the `GT_OBJ` in the usual case, though the `GT_FIELD_LIST` case may require summary info (minimally a bool indicating whether it has any GC ptrs).